website/screens/utilities/halstackProvider/examples/customTranslations.tsx

```diff
@@ -0,0 +1,65 @@
+import {
+  DxcTextInput,
+  DxcPaginator,
+  DxcFileInput,
+  HalstackProvider,
+  DxcInset,
+} from "@dxc-technology/halstack-react";
+import { useState } from "react";
+
+const code = `() => {
+    const [items, changeItems] = useState(10);
+    const [files, setFiles] = useState([]);
+
+    const itemsPerPageClick = (value) => {
+      changeItems(value);
+    };
+    const callbackFile = (files) => {
+      const updatedFiles = files.map((file) => {
+        return { ...file };
+      });
+      setFiles(updatedFiles);
+    };
+  
+    const labels = {
+      formFields: {
+        optionalLabel: "(Optionnel)",
+      },
+      fileInput: {
+        fileSizeLessThanErrorMessage: "La taille du fichier doit être inférieure à la taille maximale."
+      }
+    };
+  
+    return (
+      <HalstackProvider labels={labels}>
+        <DxcInset space="2rem">
```

Comment:
We are not properly using our Layout components here:

```
import {
  DxcTextInput,
  DxcFileInput,
  HalstackProvider,
  DxcInset,
  DxcParagraph,
  DxcFlex,
} from "@dxc-technology/halstack-react";
import { useState } from "react";

const code = `() => {
  const [files, setFiles] = useState([]);

  const callbackFile = (files) => {
    const updatedFiles = files.map((file) => {
      return { ...file };
    });
    setFiles(updatedFiles);
  };

  const labels = {
    formFields: {
      optionalLabel: "(Optionnel)",
    },
    fileInput: {
      fileSizeLessThanErrorMessage:
        "La taille du fichier doit être inférieure à la taille maximale.",
    },
  };

  return (
    <HalstackProvider labels={labels}>
      <DxcInset space="2rem">
        <DxcFlex direction="column" gap="3rem">
          <DxcTextInput
            label="Input text"
            defaultValue="Example text"
            clearable
            optional
          />
          <DxcFlex direction="column" gap="1rem">
            <DxcParagraph>
              To see the translated error, add any file using the File Input.
            </DxcParagraph>
            <DxcFileInput
              label="Select your files"
              value={files}
              callbackFile={callbackFile}
              minSize={0}
              maxSize={0}
            />
          </DxcFlex>
        </DxcFlex>
      </DxcInset>
    </HalstackProvider>
  );
}`;

const scope = {
  DxcTextInput,
  DxcFileInput,
  HalstackProvider,
  DxcInset,
  DxcFlex,
  DxcParagraph,
  useState,
};

export default { code, scope };
```

Also, we need to clean up some unused imports and methods here (maybe you started translating the Paginator and forgot to remove it).

Reply:
Done.